src/env.cc

```diff
@@ -0,0 +1,57 @@
+#include <env.h>
+#include <env-inl.h>
+#include <v8.h>
+
+using v8::HandleScope;
+using v8::Local;
+using v8::Message;
+using v8::StackFrame;
+using v8::StackTrace;
```

Comment:
Three teeny tiny nits:

1. Can you use `#include "foo.h"` for project-local includes?

2. Can you `#include <stdio.h>` explicitly?

3. Can you move the `using` directives inside the namespace for consistency?